### DISCUSSION TITLE
Is there any way to do bulk evaluation of structures without the CLI?

### DISCUSSION THREAD

**Discussion by CompRhys** [ACEsuit/mace#253] [Q&A] [answered]
Ideally looking at some interface like the MACECalculator that would let me pass much larger batches of structures at a time such that I can do analysis of large dataset within colab. In principal I can do the io and then use the CLI from within colab but it would be nicer to have a scripting interface or example of such an interface if already implemented!

**Comment by davkovacs** [comment, marked as answer]
Hi! Isn’t the eval_configs.py what you are looking for? I think it is probably easiest to use that. It should work in colab.
